.release-please-manifest.json

```diff
@@ -1,18 +1,18 @@
 {
-  ".": "1.11.0",
-  "packages/api-explorer": "0.9.15",
-  "packages/code-editor": "0.1.7",
-  "packages/extension-api-explorer": "21.11.0",
-  "packages/extension-sdk": "21.10.1",
-  "packages/extension-sdk-react": "21.10.1",
-  "packages/hackathon": "21.10.1",
-  "packages/run-it": "0.9.15",
-  "packages/sdk": "21.10.1",
-  "packages/sdk-codegen": "21.0.22",
-  "packages/sdk-codegen-scripts": "21.0.23",
+  ".": "1.12.0",
+  "packages/api-explorer": "0.9.16",
+  "packages/code-editor": "0.1.8",
+  "packages/extension-api-explorer": "21.11.1",
+  "packages/extension-sdk": "21.10.2",
+  "packages/extension-sdk-react": "21.10.2",
+  "packages/hackathon": "21.10.2",
+  "packages/run-it": "0.9.16",
+  "packages/sdk": "21.10.2",
+  "packages/sdk-codegen": "21.0.23",
+  "packages/sdk-codegen-scripts": "21.0.24",
   "packages/sdk-codegen-utils": "21.0.11",
-  "packages/sdk-node": "21.10.1",
-  "packages/sdk-rtl": "21.0.18",
-  "packages/wholly-sheet": "0.5.13",
-  "python": "21.10.0"
+  "packages/sdk-node": "21.10.2",
+  "packages/sdk-rtl": "21.0.19",
```

Comment:
it just so happened that the user facing commits for `packages/sdk-*` were all coincidentally `fix:` so they only bumped the patch. but a future `feat:` will bump the minor for these packages going forward.

Reply:
Can you clarify .. does that mean `sdk` and `sdk-node` could get out of sync w/ Looker release version?